iP/src/duke/Deadline.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline is one type of tasks.
+ * It contains deadline for the task.
+ */
+public class Deadline extends Task{
+
+    protected LocalDate deadline;
+
+    /**
+     * constructor of Deadline
```

Comment:
Perhaps capitalize the beginning of JavaDoc comments? This applies to all other JavaDoc comments as well.